tests/profile_speed.py

```diff
@@ -0,0 +1,113 @@
+"""
+This is a profiling script to check the performance of
+Scaper. It generates 100 soundscapes in sequence 
+(no parallelization). Running it on 2019 Macbook Pro
+currently takes 158.68 seconds (02:38).
```

Comment:
It would be nice to keep track of performance a little more formally, e.g. a table with hardware specs, scaper version, and runtime.

Any thoughts about where such a table could/should live? Here in source as a comment? In a GH issue?

Reply:
I'm thinking we can put it in #105 for now. Was going to update that issue with all the relevant details as we progress.

Reply:
Or better yet, the code could automatically compile all the relevant info and print out the result or even directly add it to a CSV file which we could include in the repo?